tests/unit/specs/utils/utils.spec.js

```diff
@@ -1,4 +1,4 @@
-import { roundObjectPercentages } from "utils"
+import { roundObjectPercentages } from "src/utils"
```

Comment:
i wish this wasn't called `utils` 😝

Reply:
How about `misc`? 😄 

It's only the one function and we could move it to `num.js`

Reply:
if it's only one function, yeah let's move it!  (can be done later - i'm just being nit picky)